website/views/pages/homepage.ejs

```diff
@@ -160,7 +160,7 @@
           class="btn btn-block btn-lg btn-white pl-0 m-0"
           purpose="animated-arrow-button"
           target="_blank"
-          href="https://calendly.com/fleetdm/demo"
+          href="https://calendly.com/fleetdm/demo?utm_source=live+qa"
```

Comment:
Can we change the `utm_source` to be something more specific/relevant? or are we configuring that elsewhere?
```suggestion
          href="https://calendly.com/fleetdm/demo?utm_source=homepage+demo+top"
```

Reply:
There are two Calendly links on the homepage, the second is on [line 295].(https://github.com/fleetdm/fleet/blob/DrewBakerfdm-patch-1/website/views/pages/homepage.ejs#L295)